project/AppDependencies.scala

```diff
@@ -6,14 +6,14 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.64.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.12.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.94.0-play-28",
     "uk.gov.hmrc"             %% "play-frontend-govuk"        % "0.71.0-play-28",
-    "uk.gov.hmrc"             %% "play-language"              % "5.0.0-play-28"
+    "uk.gov.hmrc"             %% "play-language"              % "5.1.0-play-28"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.3.0"            % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.12.0"            % "test, it",
     "org.scalatest"           %% "scalatest"                      % "3.0.9"            % "test, it",
```

Comment:
scalatest has a new version: 3.2.9

Reply:
The catalogue doesn't mention an update for scalatest, is there anywhere else that shows dependency updates?

Reply:
Yeah - it's really annoying to get to.
https://mvnrepository.com/artifact/org.scalatest/scalatest

Reply:
Awesome, thanks